launch/dynamo-run/src/main.rs

```diff
@@ -116,12 +127,15 @@ async fn wrapper(runtime: dynamo_runtime::Runtime) -> anyhow::Result<()> {
     let mut in_opt = None;
     let mut out_opt = None;
     let args: Vec<String> = env::args().skip(1).collect();
-    if args.is_empty() || args[0] == "-h" || args[0] == "--help" {
+    if args.is_empty() 
+        || args[0] == "-h" 
+        || args[0] == "--help"
+        || (args.iter().all(|arg| arg == "-v" || arg == "-vv")) 
```

Comment:
Does this make `dynamo-run -v` behave the same as `-h`? If yes why?

Reply:
Yes, because other required options are missing.

Reply:
Good point.